lib/graphql/schemaV2.graphql

```diff
@@ -3496,6 +3497,7 @@ enum PaymentMethodType {
   us_bank_account @deprecated(reason: "Please use uppercase values")
   sepa_debit @deprecated(reason: "Please use uppercase values")
   bacs_debit @deprecated(reason: "Please use uppercase values")
+  bancontact @deprecated(reason: "Please use uppercase values")
```

Comment:
No need to add a new deprecated value.
```suggestion
```

Reply:
This is auto generated from the enum, the enum itself is deprecated